solsys_code/forms.py

```diff
@@ -0,0 +1,35 @@
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import ButtonHolder, Column, Layout, Row, Submit
+from django import forms
+from django.urls import reverse
+
+
+class EphemerisForm(forms.Form):
+    """
+    This form is for requesting an ephemeris of a Target object
+    """
+
+    target_id = forms.IntegerField(required=True, widget=forms.HiddenInput())
+    start_date = forms.DateTimeField(required=True)
+    end_date = forms.DateTimeField(required=True)
+    step = forms.CharField(required=False, initial='1d')
+    site_code = forms.CharField(required=True, max_length=3)
```

Comment:
This should also be a choice field. You can add help_text that includes a link to the observatory creation page if their observatory isn't listed.

Reply:
I might also allow this to be blank and default to 500. for a lot of objects you might not really care about the where and just need an estimate.

Reply:
While supporting '500' would be nice, currently this explodes in what is essentially `sorcha`:
```
  File "/home/tlister/git/fomo/solsys_code/views.py", line 736, in get
    r_obs[idx], v_obs[idx] = barycentricObservatoryRates(et_i, obscode, observatories=observatories)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tlister/venv/fomo311_venv/lib64/python3.11/site-packages/sorcha/ephemeris/simulation_geometry.py", line 213, in barycentricObservatoryRates
    mVec = np.dot(m, obsVec) * Rearth
           ^^^^^^^^^^^^^^^^^
TypeError: unsupported operand type(s) for *: 'float' and 'NoneType'
```
since there's no geocenter->obs. site vectors returned. May or may not be easy to fix...